### DISCUSSION TITLE
plaintell command

### DISCUSSION THREAD

**Discussion by SirDoof** [Zrips/CMI#7134] [Feature Requests]
I would love to see a `/plaintell` command that works similar to to Minecraft's `/tellraw` but without needing to write out in json formatting.
It would support color codes + hex.

Command: `/plaintell <player> <message>`

**Comment by mrfloris** [comment]
There is ctellraw;

`[ctellraw [playerName/all] [formattedMessage]](https://www.zrips.net/cmi/commands/)`
**Send tellraw type message**
format is like cmi's custom text: https://www.zrips.net/cmi/custom-text/

```
/cmi ctellraw player <T>text, {#red}hover over me!&r</T><H>You did great, now {#gold}click me&r</H><CC>cmi heal [playerName]</CC>
```

<img width="1003" alt="Screenshot 2022-08-20 at 13 18 26" src="https://user-images.githubusercontent.com/28841349/185743528-7253e2c6-c6ce-460e-9f1a-64f41eb3bcf3.png">
